api/api.proto

```diff
@@ -2512,6 +2512,25 @@ service OpenStorageVolume {
           body: "*"
         };
     }
+
+
+}
+
+
+// OpenStorageWatcher is a service used to stream events in the system
```

Comment:
nit: OpenStorageWatcher is a service that provides APIs for watching on resources and receive them as a stream of events.